src/main/java/com/ProjektInzynierski/BackEnd/util/ResultMap.java

```diff
@@ -38,6 +38,12 @@
         return resultMap;
     }
 
+    public Map<String, String> createPasswordResetSuccessMap(String result) {
+        Map<String, String> resultMap = new HashMap<>();
+        resultMap.put(RESULT, result);
+        return resultMap;
+    }
+
```

Comment:
Czy to nie to samo co createSuccessMap?

Reply:
a tak, masz rację. Jakoś tego nie zauważyłem